tests/test_templates.py

```diff
@@ -281,6 +289,7 @@ def test_integration_qubit_diffable(self, template1, diffable1, nondiffable1,
                                         template2, diffable2, nondiffable2,
                                         interface, to_var):
         """Tests integration of qubit templates passing differentiable arguments as positional arguments to qnode."""
+        #TODO: rewrite test to avoid quadratic growth of test cases with the number of templates
```

Comment:
This could be the topic of a corresponding issue as well :)

Reply:
Added this as an issue.